scene/gui/rich_text_label.cpp

```diff
@@ -815,7 +842,36 @@ void RichTextLabel::_gui_input(Ref<InputEvent> p_event) {
 							}
 						}
 					}
+				} else if (b->is_pressed() && b->is_doubleclick() && selection.enabled) {
 
+					//doubleclick: select world
```

Comment:
Not sure if I want to select a world in my RichTextLabel. :P

Reply:
Well, sometimes a word's a world :-) Indeed I copied this over from `TextEdit`. Corrected for both comments now.